### PR TITLE
Reduces the amount of money cargo gets for coffins

### PR DESCRIPTION
**Pull request by mcmeiler**
## About The Pull Request
Title

## Why It's Good For The Game
Getting free money from cargo is not a great concept, especially when crates can be stacked via disposals and only take up 1 turf on the shuttle. This can get out of hand really quickly with enough money spent on wood.

## Testing Photographs and Procedure

<details>
This is a 1 line value change, is this really necessary?
<summary>Screenshots&Videos</summary>
I'll put some later I guess?
</details>

## Changelog
:cl: mc_meiler
tweak: Coffins now sell for 140 instead of 250 credits.
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
Could just make them take time to build so you can make money if you put effort into it

**Comment by mcmeiler** [comment]
Checked the crate price and it's actually 1700 credits for the crate.

1700 cost from the box, 50/5*200 = 2000 from coffins + 500 for the shipment box itself + 100 for a stamped manifest = 2600 credits turnover, meaning you still make 900 (!!) credits per crate of wood you turn into coffins.

File: code/modules/cargo/exports/large_objects.dm [hunk 1]
```diff
@@ -27,7 +27,7 @@
 	exclude_types = list()
 
 /datum/export/large/crate/coffin
-	cost = 250//50 wooden crates cost 2000 points, and you can make 10 coffins in seconds with those planks. Each coffin selling for 250 means you can make a net gain of 500 points for wasting your time making coffins.
+	cost = 200 //50 wood costs 2000, makes 10 coffins, makes 2000 back. No free money allowed, considering they can be easlily stacked with disposal loops
```
Comment (mcmeiler) [inline code/modules/cargo/exports/large_objects.dm]:
```suggestion
	cost = 100 //50 wood costs 2000, makes 10 coffins, makes 1000 back. No free money allowed, considering they can be easlily stacked with disposal loops. Additionally you still get 600 credits from the box + manifest either way, for a total of 1600 back.
```

**Comment by mcmeiler** [comment]
> Could just make them take time to build so you can make money if you put effort into it

Once again, time isn't something that should gate literally infinite money. IMO this shouldn't happen at all, not even with a time requirement behind it.

**Comment by PowerfulBacon** [comment]
> Once again, time isn't something that should gate literally infinite money. IMO this shouldn't happen at all, not even with a time requirement behind it.

Real life factories when they use the infinite money glitch by turning raw resources into something more valuable with only the only thing that stops us all in the way, time.

**Comment by FortSumter** [comment]
A good question is exactly how capitalist is the station meant to be? We have vendors you have to spend money at but the cook and bar work hourly and on tips . There's also lathes that anyone can use that print a lot of things for basically free.

We are basically communists.

**Comment by mcmeiler** [comment]
> Real life factories when they use the infinite money glitch by turning raw resources into something more valuable with only the only thing that stops us all in the way, time.

Real life factories when they have less than 5 seconds of processing time from raw to finished only to sell their end product for $1000+ profit.

Real life industrial companies have hours, if not *days* of processing time for *single steps in the line of production*.

If you call the few seconds required to build 10 coffins enough for over 1000 credits profit, close this PR, because I honestly can't argue with *that* kind of logic. Increasing the time on construction also won't do jack shit, as players tend to be sweatshop workers.

**Comment by FortSumter** [comment]
>  as players tend to be sweatshop workers.

Gee, it's almost as if most jobs on the station give the players nothing better to do but optimize the efficiency of doing the job by gaming the system!

As a cargo tech, my MO is typically to get the bounties done ASAP even if I have to break into places and do them myself. If other players are also gaming their departments, we'll also have trituim to sell and a shitload of plasma. We then spend the next 2 hours sitting on a 104k+ cargo budget doing absolutely nothing because no one requests any crates and there is no station wide emergency and there's nothing much interesting we can buy that isn't either restricted and/or lethal weaponry.

How about you go ruin some other departments fun, medical perhaps. It's one thing to stitch someone back together, and quite another to perform literal necromancy and bring someone back to life whos been a rotting corpse for an hour by performing simple organ surgeries.

**Comment by PowerfulBacon** [comment]
I'm suggesting:
 - Add a value to the crafting time of coffins
 - Make the profit margin somewhat reasonable for the amount of time it takes.

It's not one or the other, they can still make profit without making all the credits in the world.

**Comment by mcmeiler** [comment]
> Also wood doesn't even cost 2000, which is probably worth considering if you are basing your calculations off of them costing 200.

I've literally already addressed this in [65ae0b3](https://github.com/BeeStation/BeeStation-Hornet/pull/7331/commits/65ae0b3bc84ce2bcf016e04dc3d8d3f6fc2e4dde).
It's currently at -100 per box in total.

I'm willing to compromise to around +300 credits total, which is over a quarter of what it was before. This still fucks the economy really badly, but none of you really seem intent on fixing it any time soon anyway.

File: code/modules/cargo/exports/large_objects.dm [hunk 2]
```diff
@@ -27,7 +27,7 @@
 	exclude_types = list()
 
 /datum/export/large/crate/coffin
-	cost = 250//50 wooden crates cost 2000 points, and you can make 10 coffins in seconds with those planks. Each coffin selling for 250 means you can make a net gain of 500 points for wasting your time making coffins.
+	cost = 100 //50 wood costs 2000, makes 10 coffins, makes 1000 back. No free money allowed, considering they can be easlily stacked with disposal loops. Additionally you still get 600 credits from the box + manifest either way, for a total of 1600 back.
```
Comment (mcmeiler) [inline code/modules/cargo/exports/large_objects.dm]:
```suggestion
	cost = 140 //50 wood costs 1700, makes 10 coffins, makes 1400 back. No free money allowed, considering they can be easlily stacked with disposal loops. Additionally you still get 600 credits from the box + manifest either way, for a total of 2000 back. Total of 300 profit for wasting your time building coffins.
```

**Comment by FortSumter** [comment]
> This still fucks the economy really badly, but none of you really seem intent on fixing it any time soon anyway.

Economy? ECONOMY?! WHAT THE FUCK IS AN ECONOMY?! 

You must be joking. There is no economy in this game and if there was, no one would give a fuck outside of you coders. Like I said, the entire concept of an economy is jank when anyone can print whatever they want from lathes and cargo effectively has no purpose outside of emergency situations. The only thing people use money for is to buy things from vending machines because most normal crew never make enough money to afford even some basic crates and even if they can, many of the crates people would want are restricted.

You want to really make money matter? you want to add a realistic economy to the station? Nerfing cargo more isn't it. You will have to do a bottom-up complete rework of both cargo and how money can be made relevant in all aspects of the station.

Want some decent healthcare? better have your wallet handy. Need to be cloned? Ooooh that's gonna cost you. Want to avoid being harassed by security? a bribe will go a long way. Want even the most basic of tools? You better start saving for a cargo crate. Need your kitchen repaired and upgraded? better start getting quotes from engineers.

No more doing things for free, no more getting anything handed to you, no guarantees you will even have access to basic tools or items unless you have the money to pay for them. All money for goods or services will be paid directly to departmental budgets or to individuals, including things purchased from vending machines. Pure capitalism, baby!

**Comment by Fronsis** [comment]
Uhhh as Fort said? using the argument of ''fucking the economy'' does't.. really apply to us, we've been on track on nerfing the obvious massive cash givers a long time ago (miasma canisters farm for example) We don't really get a ''The Whole cargo crew is riding on atvs and is ordering bikes 30 minutes into the round!!!'' anymore, i've recently started playing Cargo and i'm personally more inclined to go around asking for bounties that sit and waste my time doing a cheap trick with wood crates,as Bacon said it's all about how time consuming it can get. 

I mean it's fine and makes sense to fix it since it's sort of an abuse for free money but i don't think it's also worth it to go and check if the price of every crate/item can't be converted into an scenario like the Coffin one, but considering the current economy status it's like eh

We're certainly in need of an economy rework and i hope one day we'll have a more balanced approach like TG/SR since they introduce some more roleplay and newbie friendly stuff (civilian bounty pad and the bots that harass kitchen/bar to give an example)

**Comment by mystery3525** [comment]
I adminbus remove coffins in the cargo shuttle and scold cargo via CC messages every chance I get

**Review by Miliviu — APPROVED** [review]
LGTM

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**